doc/man3/OSSL_PARAM_int.pod

```diff
@@ -399,10 +407,11 @@ L<openssl-core.h(7)>, L<OSSL_PARAM(3)>
 =head1 HISTORY
 
 These APIs were introduced in OpenSSL 3.0.
+The macro OSSL_PARAM_is_empty() was added in OpenSSL 3.5.
```

Comment:
We'd normally have a blank line betwixt these two.

Reply:
internal now - so changes were removed.